pkg/volume/downwardapi/downwardapi_test.go

```diff
@@ -363,6 +378,27 @@ func (step verifyMode) run(test *downwardAPITest) {
 	}
 }
 
+type verifyFsUser struct {
+	stepName
+	expectedFsUser int64
+}
+
+func (step verifyFsUser) run(test *downwardAPITest) {
+	fileInfo, err := os.Stat(filepath.Join(test.volumePath, step.name))
+	if err != nil {
+		test.t.Errorf(err.Error())
+		return
+	}
+	stat_t, ok := fileInfo.Sys().(*syscall.Stat_t)
+	if !ok {
+		test.t.Errorf("Couldn't get file stat_t")
+		return
+	}
+	if uid := int64(stat_t.Uid); uid != step.expectedFsUser {
+		test.t.Errorf("Found uid `%v` expected %v", uid, step.expectedFsUser)
```

Comment:
This error message doesn't give context about what broader test has failed.

Reply:
That is done from the function calling this interface. We are just adding a type, following the conventions used in this test file.